meilisearch/src/routes/api_key.rs

```diff
@@ -45,10 +49,72 @@ pub async fn create_api_key(
     Ok(HttpResponse::Created().json(res))
 }
 
+#[derive(Debug)]
+pub struct PaginationDeserrError {
+    error: String,
+    code: Code,
+}
+
```

Comment:
All of these error types will need to be factored out at some point. But for a first implementation, it's okay.